postcss.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = ctx => {
+  return {
+    parser: false,
+    map: false,
+    plugins: {
+      'postcss-import': {},
```

Comment:
Since this file is defined in the root project, we should define those dependencies in the root `package.json`